__tests__/sources.ts

```diff
@@ -132,17 +216,34 @@ describe('query sources', () => {
           flags {
             featured
           }
+          category {
+            id
+          }
         }
       }
     }
   }`;
 
   it('should return only public sources', async () => {
-    const res = await client.query(QUERY(10, true));
+    const res = await client.query(
+      QUERY({ first: 10, filterOpenSquads: true }),
+    );
     const isPublic = res.data.sources.edges.every(({ node }) => !!node.public);
     expect(isPublic).toBeTruthy();
   });
 
+  it('should filter by category', async () => {
+    console.log('entry');
+    const repo = con.getRepository(Source);
+    await repo.update({ id: 'a' }, { categoryId: 'general' });
+    await repo.update({ id: 'b' }, { categoryId: 'web' });
+    const res = await client.query(QUERY({ first: 10, categoryId: 'web' }));
```

Comment:
Do we allow filtering on more categories at once?
(If so we'll need a test, if only 1 it's fine as is)

Reply:
At least so far from product perspective looks like we filter only by one or get all.

Reply:
Yeah, for now, we only do 1.